uber/badge_funcs.py

```diff
@@ -14,6 +14,15 @@ def check_range(badge_num, badge_type):
                 return '{} badge numbers must fall within the range {} - {}'.format(dict(c.BADGE_OPTS)[badge_type], min_num, max_num)
 
 
+def is_badge_the_same(attendee, old_badge_type, old_badge_num):
```

Comment:
It's a common convention for functions that start with `is_` to return a boolean. This function is interesting because  it certainly checks if the badge is the same, and it conditionally returns either a user facing message (truthy), or `None` (falsey).

Further down in this file there's this line, which relies on the truthy/falsey return values:
```python
message = is_badge_the_same(attendee, old_badge_type, old_badge_num) or "Badge updated."
```

But his function also has the side effect of setting `attendee.badge_num` if it's not already set. So it's not strictly a true/false test if the badge is the same.

This is kinda nit-picky, because the code works as is. But if I were going to make a suggestion, I'd say break this into two functions like so:
```python
def is_badge_empty_or_unchanged(attendee, old_badge_type, old_badge_num):
    old_badge_num = int(old_badge_num or 0) or None
    return old_badge_type == attendee.badge_type and \
            (not attendee.badge_num or old_badge_num == attendee.badge_num)

def set_badge_if_empty(attendee, old_badge_type, old_badge_num):
    if is_badge_empty_or_unchanged(attendee, old_badge_type, old_badge_num)
        attendee.badge_num = old_badge_num
        return 'Attendee is already {} with badge {}'.format(c.BADGES[old_badge_type], old_badge_num)
```